src/api.ts

```diff
@@ -220,8 +226,8 @@ export async function startContainer( commitHash: CommitHash, env: RunEnv ) {
 	return startPromise;
 }
 
-export async function refreshRunningContainers() {
-	const containers = await docker.listContainers();
+export async function refreshContainers() {
```

Comment:
`refreshContainers()` now loads all containers, not only running containers. I searched for all existing usages of `state.containers` and ensure all them are filtering by `container.State === 'running'` so the behaviour doesn't change.